dll/shellext/acppage/lang/cs-CZ.rc

```diff
@@ -35,13 +38,13 @@ BEGIN
     PUSHBUTTON      "<",IDC_REMOVE,144,60,18,14,WS_DISABLED
     PUSHBUTTON      "&OK",IDOK,174,114,60,14
     PUSHBUTTON      "&Storno",IDCANCEL,240,114,60,14
-    LTEXT           "Available modes",-1,6,6,52,8
-    LTEXT           "Selected modes",-1,168,6,51,8
+    LTEXT           "Dostupné režimy",-1,6,6,52,8
```

Comment:
Does that one fit the 52 width? Any pic?

Reply:
I've changed the width:
![Screenshot_live_test_2024-07-11_21:35:52](https://github.com/user-attachments/assets/8378d7f4-f998-454e-bf5f-bd91b0606dd9)

Reply:
thanks